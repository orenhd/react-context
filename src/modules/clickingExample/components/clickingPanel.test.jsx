import React from 'react';
import { shallow } from 'enzyme';

import ClickingPanel from './clickingPanel';

describe('clickingPanel', () => {
  it('test', () => {
    expect(true).toBeTruthy();
  })

  it('should render correctly', () => {
    const component = shallow(<ClickingPanel
      clickingData={{}}
      homeButtonClickedHandler={jest.fn()}
      homeButtonClickedOutsideHandler={jest.fn()}
    />);

    expect(component).toMatchSnapshot();
  });
});
