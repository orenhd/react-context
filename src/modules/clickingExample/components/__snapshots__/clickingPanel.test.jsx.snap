// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`clickingPanel should render correctly 1`] = `
<div
  className="clicking-panel"
>
  <div
    className="undefined margined-content"
  >
    <FloatingActionButton
      disabled={false}
      mini={true}
      onClick={[Function]}
      secondary={false}
      zDepth={2}
    >
      <FontIcon
        className="material-icons"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        home
      </FontIcon>
    </FloatingActionButton>
  </div>
</div>
`;
